app/views/account/pages/_rating.html.slim

```diff
@@ -0,0 +1,5 @@
+- if page.rating_progress <= 50
+  - style = 'danger'
+- else
+  - style = 'success'
+h5.t-rating class=style id=page.id = "#{page.rating} / #{page.max_rating}"
```

Comment:
what is this `id=page.id` ? 
go ahead and come up with something better and readable